code-examples.rst

```diff
@@ -121,7 +120,7 @@ Inserting a single entry
 			  method: 'POST',
 			  headers: {
 				'Content-Type': 'application/json; charset=utf-8',
-				'Content-Length': data.length
+				'Application-Id': application.id,
```

Comment:
applicationId. It's unlikely that they have a separate application object, it would rather be a configured value